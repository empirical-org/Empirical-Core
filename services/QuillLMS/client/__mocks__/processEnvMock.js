export default {
  env: {
    PUSHER_KEY: 'pusher',
    PUSHER_CLUSTER: 'cluster'
  }
}
