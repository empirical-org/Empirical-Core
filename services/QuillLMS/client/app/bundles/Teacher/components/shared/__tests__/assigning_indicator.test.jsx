import React from 'react';
import { shallow } from 'enzyme';

import AssigningIndicator from '../button_loading_indicator';

import processEnvMock from '../../../../../../__mocks__/processEnvMock.js';
window.import.meta.env.VITE_PROCESS_ENV_CDN_URL = processEnvMock.env.CDN_URL;

describe('AssigningIndicator component', () => {

  it('should render', () => {
    const wrapper = shallow(
      <AssigningIndicator />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
