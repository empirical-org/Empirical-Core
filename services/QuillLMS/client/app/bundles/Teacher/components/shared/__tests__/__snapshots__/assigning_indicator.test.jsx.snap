// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssigningIndicator component should render 1`] = `
<span
  className="assigner-container"
>
  <img
    alt="a circle being retraced"
    className="assigner"
    src="undefined/images/shared/assigner_still.png"
  />
</span>
`;
